packages/core/src/components/AccordionGroup/types.ts

```diff
@@ -0,0 +1,45 @@
+import { SerializedStyles } from '@emotion/react';
+import { CommonProps } from '@global-types/emotion';
+
+export type AccordionVariant = 'empty' | 'small' | 'medium' | 'large';
```

Comment:
Let's use `size` here and as the prop name, because in other components `variant` means a different characteristic, e.g. primary / secondary / tertiary for <Button />.

Reply:
Done